plugins.js

```diff
@@ -43,7 +43,15 @@ module.exports = ( ) => {
         It will load from ${pl.serveFrom + '-dupe'} instead.` )
       pl.serveFrom += '-dupe'
     }
-    pl.canonicalUrl = process.env.CANONICAL_URL + pl.serveFrom
+    //trim extra leading slash from serveFrom if necessary
+    const canonicalUrl = process.env.CANONICAL_URL
```

Comment:
it's best to use `new URL( ... ).origin` - it's a good spot, i'll fiddle with this now